frontend/packages/dev-console/src/components/pipelines/PipelineRuns.tsx

```diff
@@ -6,17 +6,18 @@ import {
   pipelineRunFilterReducer,
   pipelineRunStatusFilter,
 } from '../../utils/pipeline-filter-reducer';
+import { ListFilterId, ListFilterLabels } from '../../utils/pipeline-utils';
 import { PipelineRunModel } from '../../models';
 
-const filters = [
+export const runFilters = [
   {
     type: 'pipelinerun-status',
-    selected: ['Succeeded'],
+    selected: [ListFilterId.Succeeded, ListFilterId.Failed, ListFilterId.Running],
     reducer: pipelineRunFilterReducer,
     items: [
-      { id: 'Succeeded', title: 'Complete' },
-      { id: 'Failed', title: 'Failed' },
-      { id: 'Running', title: 'Running' },
+      { id: ListFilterId.Succeeded, title: ListFilterLabels[ListFilterId.Succeeded] },
+      { id: ListFilterId.Failed, title: ListFilterLabels[ListFilterId.Failed] },
+      { id: ListFilterId.Running, title: ListFilterLabels[ListFilterId.Running] },
```

Comment:
The order is not consistent. Perhaps we make `items` a variable somewhere else? We'd want the filter items to be in the same order throughout the pipelines otherwise the user instinctively won't be able to click on filters without stopping to read what order they happen to be in on a given page.

Reply:
Sorry I missed this on my previous review passes.

Reply:
Updated to have a consistent order throughout